.github/scripts/sync_to_dify.py

```
@@ -0,0 +1,96 @@
+        ["git", "diff", "--name-only", "HEAD~1", "HEAD"],
+        capture_output=True, text=True
+    )
+    files = result.stdout.strip().splitlines()
+    return [f for f in files if f.endswith(".md") and os.path.exists(f)]
+
+
+def get_existing_documents():
+    docs = {}
+    page = 1
+    while True:
+        resp = requests.get(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)